package.json

```diff
@@ -4,15 +4,15 @@
     "version": "0.0.0",
     "license": "UNLICENSED",
     "devDependencies": {
-        "@types/node": "^10.14.8",
-        "@types/shelljs": "^0.8.8",
-        "concurrently": "^5.3.0",
-        "envfile": "^2.1.1",
-        "json": "^9.0.6",
-        "lerna": "^2.5.1",
-        "rimraf": "^2.6.2",
-        "ts-node": "<7.0.0",
-        "typescript": "~4.1.2"
+        "@types/node": "^16.7.8",
+        "@types/shelljs": "^0.8.9",
+        "concurrently": "^6.2.1",
+        "envfile": "^6.17.0",
+        "json": "^11.0.0",
+        "lerna": "^4.0.0",
+        "rimraf": "^3.0.2",
+        "ts-node": "^10.2.1",
+        "typescript": "^4.4.2"
```

Comment:
(off-topic) we should review whether we need resolutions still, it can introduce api incompatibilities and pull dependencies which we don't use anymore